cli/main.rs

```diff
@@ -204,11 +204,12 @@ pub fn create_main_worker(
     no_color: !colors::use_color(),
     get_error_class_fn: Some(&crate::errors::get_error_class_name),
     location: program_state.flags.location.clone(),
-    location_data_dir: program_state.flags.location.clone().map(|loc| {
+    origin_data_dir: program_state.flags.location.clone().map(|loc| {
       program_state
         .dir
         .root
         .clone()
+        // TODO(@crowlKats): change to origin_data for 2.0
         .join("location_data")
```

Comment:
Do we need to consider this breaking? I feel like we provide cache paths in `deno info` so we can rename them while providing a stable handle, and this wasn't even in `deno info` yet.

Reply:
well renaming `location_data` would mean that users that have used local storage would lose their current data

Reply:
I think we can still do this in 1.x - we add a "migration" that renames old to new. Lets do this in another PR though.